packages/grafana-ui/src/components/uPlot/config/UPlotAxisBuilder.ts

```diff
@@ -62,6 +62,10 @@ export class UPlotAxisBuilder extends PlotConfigBuilder<AxisProps, Axis> {
 
     // for axis left & right
     if (axis.side !== 2 || !scale) {
+      if (plotDim <= 150) {
+        return 15;
+      }
+
       return 30;
```

Comment:
Can we remove the magic of these numbers and add them as const, similar to `UPLOT_AXIS_FONT_SIZE`?

Reply:
done